### ISSUE TITLE
awaitMessage issue

### ISSUE THREAD

**Issue by kyranet** [UnseenFaith/komada#158]
### Version of Komada
0.18.3#indev

### Are core files modified?
No

### Describe the problem
```
TypeError: client.funcs.permissionLevel(...).catch is not a function
    at Object.module.exports [as awaitMessage] (/mnt/c/Users/manue/node_modules/komada/functions/awaitMessage.js:8:90)
    at module.exports (/mnt/c/Users/manue/functions/handleCommand.js:17:22)
```

### Expected Behaviour
If `client.config.cmdPrompt` is set to true, it should prompt (even thought it's broken until the usage.js rewrite).

### Actual Behaviour
It logs on the console as an uncaught promise error, with no new message on the channel.

### Steps to Reproduce.
Download `0.18.3#indev` and set `cmdPrompt` to `true`.

**Comment by UnseenFaith** [comment]
This is because permLevel was changed. awaitMessage needs to be changed to use either
`msg.author.permLevel` or `msg.member.permLevel` depending on the circumstance.

**Comment by kyranet** [comment]
And also the function permissionLevel does no longer return a promise. So `.catch()` is not a function for it. Removing it will fix this issue, but I'll leave it for until you fix the prompts after usage rewrite.

**Comment by CyberiumShadow** [comment]
Fixed in #163
